Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1339,6 +1367,11 @@ extension BrowserViewController : Transitionable {
             }
         }
         self.homePanelController?.view.hidden = false
+        if options.toView === self {
+            startTrackingAccessibilityStatus()
+        } else {
+            stopTrackingAccessibilityStatus()
+        }
```

Comment:
What I meant was having some additional flag that could be set programmatically (e.g., via a unit test) that would enable the `AuralProgressBar` feedback. Of course, KIF can't test for audio, so we'd have to do something like fire notifications when the `progress`/`hidden` states are updated. Or we could figure out a way for KIF to get a direct reference to the `AuralProgressBar` so it can verify the properties as they change.